.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+node_js:
+  - 'stable'
+cache:
+  directories:
+    - 'node_modules'
+install:
+  - npm install
+before_script:
+  - npm install codeclimate-test-reporter -g
```

Comment:
We decided not to use code climate for coverage report, we are using coveralls. please fix this.